sources/packages/backend/package.json

```diff
@@ -33,9 +33,12 @@
     "test:e2e:workflow:local": "cross-env ENVIRONMENT=test TZ=UTC jest --verbose --config ./workflow/test/jest-e2e.json --forceExit",
     "test:e2e:queue-consumers": "npm run db:seed:test filter=CreateCASDistributionAccounts && cross-env ENVIRONMENT=test jest --collect-coverage --verbose --config ./apps/queue-consumers/test/jest-e2e.json --runInBand --forceExit",
     "test:e2e:queue-consumers:local": "cross-env ENVIRONMENT=test TZ=UTC jest --verbose --config ./apps/queue-consumers/test/jest-e2e.json --runInBand --forceExit",
-    "migration:run": "node --abort-on-uncaught-exception --require ts-node/register apps/db-migrations/src/main.ts",
+    "migration": "ts-node -r tsconfig-paths/register --transpile-only apps/db-migrations/src/main.ts",
```

Comment:
is `"start:prod:db-migrations"` required anymore in this file?

Reply:
Nope, I will remove it.